README.rst

```diff
@@ -0,0 +1,64 @@
+|BuildStatus|_ |TotalDownloads|_ |LatestStableVersion|_ |License|_ |TYPO3|_
+
+.. |BuildStatus| image:: https://github.com/FriendsOfTYPO3/extension_builder/workflows/tests/badge.svg
+   :alt: Build Status
+.. _BuildStatus: https://github.com/FriendsOfTYPO3/extension_builder/actions
+
+.. |TotalDownloads| image:: https://poser.pugx.org/friendsoftypo3/extension-builder/d/total.svg
+   :alt: Total Downloads
+.. _TotalDownloads: https://packagist.org/packages/friendsoftypo3/extension-builder
+
+.. |LatestStableVersion| image:: https://poser.pugx.org/friendsoftypo3/extension-builder/v/stable.svg
+   :alt: Latest Stable Version
+.. _LatestStableVersion: https://packagist.org/packages/friendsoftypo3/extension-builder
+
+.. |License| image:: https://poser.pugx.org/friendsoftypo3/extension-builder/license.svg
+   :alt: License
+.. _License: https://packagist.org/packages/friendsoftypo3/extension-builder
+
+.. |TYPO3| image:: https://img.shields.io/badge/TYPO3-11-orange.svg
+   :alt: TYPO3
+.. _TYPO3: https://get.typo3.org/version/11
+
+=====================================
+TYPO3 Extension ``extension_builder``
+=====================================
+
+The *Extension Builder* helps you to develop a TYPO3 extension based on the
+domain-driven MVC framework `Extbase <https://docs.typo3.org/m/typo3/book-extbasefluid/master/en-us/0-Introduction/Index.html>`__
+and the templating engine `Fluid <https://docs.typo3.org/m/typo3/book-extbasefluid/master/en-us/8-Fluid/Index.html>`__.
+It provides a graphical editor to specify and connect domain models and
+generates boiler plate code that can be installed and further developed.
+
+:Repository:  https://github.com/FriendsOfTYPO3/extension_builder
+:Read online: https://docs.typo3.org/p/friendsoftypo3/extension-builder/master/en-us/
+
+**TODO: The following text should be moved into the documentation.**
```

Comment:
I would like to see the composer installation command here because that is the most important information and I don't want to go to the documentation first in order to know what is the composer name.

And this todo should be a commend and not be visible

Reply:
Simply suggest your change.